sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -2596,4 +2596,33 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
           Date.valueOf("2017-02-12")))
     }
   }
+
+  test("SPARK-41235: Array Compact") {
```

Comment:
This is not a bug fix, so it is not necessary to add `SPARK-41235` to the test case name